src/app/credExplorer/pagerankTable/TableRow.js

```diff
@@ -0,0 +1,74 @@
+// @flow
+
+import React, {type Node as ReactNode} from "react";
+
+type TableRowProps = {|
+  +depth: number,
+  // The node that goes in the Description column
+  +description: ReactNode,
+  // What proportion should be formatted in the connection column
+  +connectionProportion: ?number,
+  // The score to format and display
+  +score: number,
+  // Children to show when the row is expanded
+  +children: ReactNode,
+|};
+type TableRowState = {|
+  expanded: boolean,
+|};
+
+export function scoreDisplay(score: number) {
+  return score.toFixed(2);
+}
+
+export class TableRow extends React.PureComponent<
```

Comment:
Okay. I like this component. The name leaves something to be desired—why
is a `TableRow` different from a `tr`?—but given that it’s an
implementation detail of the `pagerankTable` module, I’m okay with it.

Reply:
Acknowledged.